src/appshell/qml/DevTools/Gallery/GeneralComponentsGallery.qml

```diff
@@ -389,6 +390,43 @@ Rectangle {
         }
     }
 
+    Component {
+        id: visibleButtonSample
+
+        Column {
+            spacing: 8
+
+            width: 200
+
+            VisibleButton {
```

Comment:
it might make sense to rename it onto VisibilityButton or just an EyeButton

Reply:
agree with you, I forgot to rename to VisibilityButton